CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Changelog
 =========
 
-Version 1.2.2 (23 May, 2025)
+Version 1.2.3 (23 May, 2025)
```

Comment:
i needed to bump again bc of a signing issue that occurred btwn closing and releasing steps. some artifacts were released at 1.2.2, and sonatype didn't allow me to overwrite.